server/src/test/java/org/opensearch/cluster/coordination/NodeJoinTests.java

```diff
@@ -213,7 +213,7 @@ protected void onSendRequest(long requestId, String action, TransportRequest req
                         requestId,
                         new TransportService.HandshakeResponse(destination, initialState.getClusterName(), destination.getVersion())
                     );
-                } else if (action.equals(JoinHelper.VALIDATE_JOIN_ACTION_NAME)) {
```

Comment:
Please update and keep tests for both actions.

Reply:
Keeping it as it is and adding VALIDATE_COMPRESSED_JOIN_ACTION_NAME case also.